docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -123,12 +123,12 @@ The probability of side effects is also taken from the literature, and if a pers
 BAU parameter data tables
 +++++++++++++++++++++++++
 
```

Comment:
Definitions:
"high LDL-C" in BAU means LDL-C > 4.9.
"high LDL-C" in 2 intervention scenarios is determined by CV risk SCORE, but generally can be thought of as LDL-C>3.0.
"hitting target" means a 50% or more reduction over untreated LDL-C.
"high potency statin" = atorvastatin and rosuvastatin
"low potency statin" = simvastatin, pravastatin, atorvastatin, fluvastatin
"high dose" = 40mg and up
"low dose" = under 40mg.